bylaws/009-per-workflow.md

```diff
@@ -0,0 +1,85 @@
+
+If the vote passes, the proposal officially enters Review Phase. If it does not, it remains in Draft Phase.
+
+## Review
+
+The Review Phase is an opportunity for the community to experiment with a reasonably fixed target to evaluate a proposal's practicality. At this stage, the Sponsor is the final authority on changes to the specification as well as any decisions to move the proposal forward or backward, however, the Editor may veto proposed changes they believe are harmful to the design of the specification.
+
+During this phase, trial implementations of the specification are expected and encouraged. Changes to the specification are limited to those directly informed by trial implementations, wording, typos, clarification, etc. Major changes are not permitted in this phase. If the development of trial implementations demonstrates the need for major changes then the specification must be pushed back to Draft Phase. Any incompatible change that would require significant effort for trial implementations to adjust for qualifies as a major change. Small to moderate incompatible changes do not necessarily mandate a return to Draft Phase.
+
+Unless a proposal is moved to Draft stage again, it must remain in Review stage for a minimum of four weeks and until two independent viable trial implementations can be demonstrated. The responsibility for finding such trial implementations and presenting them to the Core Committee lies with the Working Group, and especially the Editor and Sponsor. As not all specifications are PHP interfaces where the definition of "implementation" is self-evident, the Sponsor should use good faith judgement to determine when that is the case. Any member of the Core Committee may object to a given trial implementation as irrelevant or insufficient with due cause.
+
+Once four weeks have passed and two viable trial implementations can be demonstrated, the Sponsor may call an Acceptance Vote. If the Acceptance Vote fails, the specification may remain in Review.
+
+## Accepted
+
+If the Acceptance Vote passes, then the proposal officially becomes an accepted PER.  It is considered the 1.0.0 release of that PER.
+
+## Evolution
+
+Unlike a PSR, a PER is expected to evolve over time.  Once the initial version of a PER is approved, the Working Group remains active but its minimum requirements drop to an Editor and two additional individuals.  By default, the Working Group at time of first approval continues in that role.  The Working Group may continue to develop the PER over time as the context of the recommendation evolves, such as newly released PHP language versions, additional input from third parties, etc.
+
+For any proposed change to the PER, the Editor must notify the Core Committee via a post to the mailing list of an Intent to Merge a given change.
```

Comment:
Mm, possibly.  My concern is that a whole crapload of stuff could be in a given release, and then if the CC objects you have to sort out how to pull out or change selected pieces.

"Ask forgiveness, not permission" makes saying "no" much harder, even in cases where saying no is the right thing to do.  And sometimes, it will be the right thing to do.

Reply:
Well I would hope that the people most likely to have strong opinions about a project would be part of the Working Group. I am pretty sure that "notify the CC of any change" is going to be overly burdensome.